paymentsheet/src/main/java/com/stripe/android/paymentsheet/FlowControllerCompose.kt

```diff
@@ -84,6 +84,25 @@ fun rememberPaymentSheetFlowController(
     )
 }
 
+/**
+ * Creates a [PaymentSheet.FlowController] that is remembered across compositions and initializes optional
```

Comment:
> initializes optional callbacks if set

This seems like an implementation detail that we don’t need to mention.

Reply:
Updated